guides/NEW_APP.md

```diff
@@ -0,0 +1,50 @@
+Creating a new app
+==================
+
+The following steps will guide you through add adding a new app to our system that is built by the CI system and deployed to the live environment.
+
+Prerequisites
+-------------
+
+In order to follow this guide you will need:
+
+* A github account with access to the ONSdigital organisation
+
+* A login for the concourse CI environment
+
+* [dp-cli](https://github.com/ONSdigital/dp-cli)
+
+* Git:
+
+  ```
+  brew install git
+  ```
+
+* Terraform >= 0.12:
+
+  ```
+  brew install terraform
+  ```
+
+* Fly cli (can be downloaded from the links in the corner of the concourse UI)
+
+Getting started
+---------------
+
+These steps will create a new app that is built by CI and deployed to the develop and production environments.  If any of the following github links 404 it is likely that you need to be added to the ONSdigital organisation.
+
+1. Determine what language you want to use and technology your app will likely use
```

Comment:
- Bit pedantic again the sentence could be optimised a bit maybe something like
```suggestion
1. Determine what language and technologies you will use
```
- The general theme of this guide is to be informative and this list is a step-by-step set of instructions. So maybe worth avoiding words like 'want' and 'likely'.